go.mod

```diff
@@ -128,7 +129,7 @@ replace (
 
 	github.com/cosmos/ibc-go/v3 => github.com/cosmos/ibc-go/v3 v3.0.0-alpha1.0.20220510152759-dcd0681d8f07
 
-	github.com/cosmos/interchain-accounts => github.com/devashishdxt/interchain-accounts-demo v0.1.1-0.20220530041519-df4b2c11bb62
+	github.com/cosmos/interchain-accounts => github.com/devashishdxt/interchain-accounts-demo v0.1.1-0.20220705075641-407559670229
```

Comment:
maybe can use https://github.com/crypto-org-chain/interchain-accounts-demo

Reply:
Yes. Forgot about this. Will change.